packages/shared-base/src/types/SocialIdentity.ts

```diff
@@ -61,6 +61,8 @@ export interface SocialAddress<ChainId> {
     updatedAt?: string
     /** Create timestamp (unix timestamp) */
     createdAt?: string
+    /** Data from official sources indicate validation, e.g. twitter blue  */
+    verified?: boolean
```

Comment:
verifiedByTwitter?: boolean

Reply:
I think we can do without a separate field for twitter, after all, we have other data sources from other sources, and in the future there may be other sources that don't require firefly validation